server/middlewares/verifyArticleId.js

```diff
@@ -0,0 +1,31 @@
+import models from '../models';
+
+const { Article } = models;
+
+const verifyArticleId = async (req, res, next) => {
```

Comment:
I feel `verifyArticleId` is not descriptive enough. When I see it for the first time, I thought you would query for the articleId but that's not the case. `validateArticleId` sounds better to me. What do you think?

Reply:
@sulenchy Thanks for pointing that out. I agree, validate is more descriptive of what the function does.